src/yb/util/wait_state.h

```diff
@@ -85,76 +96,66 @@ YB_DEFINE_ENUM_TYPE(
-    // Reads
-    (GetSafeTime)(GetSubDoc)(DoRead)
+    ((TabletActiveOnCPU, YB_TABLET_WAIT))  // A write-rpc is acquiring the required locks.
+    (LockedBatchEntry_Lock)
+    (MVCCWaitForSafeTime)
+    (BackfillIndexWaitForAFreeSlot)
+
+    (TransactionStatusCache_DoGetCommitData)
+    (XreplCatalogManagerWaitForIsBootstrapRequired)
+    (RpcsWaitOnMutexInShutdown)
+    (TxnCoordWaitForMutexInPrepareForDeletion)
+    (PgResponseCache_Get)
+    (TxnResolveSealedStatus)
+    (PgClientSessionStartExchange)
+    (WaitForYsqlBackendsCatalogVersion)
+    (CreatingNewTablet)
 
     // OperationDriver
-    ((ExecuteAsync, YB_CONSENSUS))  // Raft request  being enqueued for preparer.
-        (PrepareAndStart) // 
-        (SubmittedToPreparer) // raft-operation has been submitted to preparer.
-        (AddedToLeader) // raft-operation written to the leader's log. Waiting for response from followers.
-        (AddedToFollower) // On the follower side.
-        (HandleFailure) // Ran into some kind of failure.
-      (Applying) // operation has been replicated/acknowledged by a majority. Applying it to the tablet.
-      (ApplyDone) // apply is done.
-
-    // UpdateConsensus
-      (Updating) // Update rpc is called on a follower
-      (UpdateReplica) // Handling UpdateReplica
-      (DoneUpdate)   // Done update
-
-    // Debugging
-    // Various specific operations in SubmittedToPrepare state
-      (SubmittedWriteToPreparer)
-      (SubmittedChangeMetadataToPreparer)
-      (SubmittedUpdateTransactionToPreparer)
-      (SubmittedSnapshotToPreparer)
-      (SubmittedTruncateToPreparer)
-      (SubmittedEmptyToPreparer)
-      (SubmittedHistoryCutoffToPreparer)
-      (SubmittedSplitToPreparer)
-      (SubmittedChangeAutoFlagsConfigToPreparer)
-      (SubmittedUnexpectedToPreparer)
+    ((RaftActiveOnCPU, YB_CONSENSUS))  // Raft request  being enqueued for preparer. -- never seen
+      (WALLogSync) // waiting for WALEdits to be persisted.
+      (RaftWaitingForQuorum)
+      (ApplyingRaftEdits)
 
     // Flush and Compaction
     ((StartFlush, YB_FLUSH_AND_COMPACTION))(StartCompaction)
-    (OpenFile)(CloseFile)(DeleteFile)(WriteToFile)
+    (OpenFile)
+    (CloseFile)
+    (DeleteFile)
+    (WriteToFile)
     (StartSubcompactionThreads)(WaitOnSubcompactionThreads)
 
     // CQL Wait Events
     ((Parse, YB_CQL_WAIT_STATE))(Analyze)(Execute)(ExecuteWaitingForCB)
-    (CQLRead)(CQLWrite)(CQLHandling)(CQLHandlingDone)(CQLFlushAsync)(CQLFlushAsyncDone)
+    (CQLActiveOnCPU)
+    (CQLRead)(CQLWrite)
+    (CQLWaitingOnDocdb)
 
-    ((RocksDB, YB_ROCKSDB))
-       (BlockCacheLookupInCache)
-       (BlockCacheLookupCompressed)
+    ((RocksDBActiveOnCPU, YB_ROCKSDB))
        (BlockCacheReadFromDisk)
 
     // Perform Wait Events
-    ((DmlRead, YB_PG_WAIT_PERFORM)) (DmlWrite) (DmlReadWrite)
+    ((DmlRead, YB_PG_WAIT_PERFORM))
+    (DmlWrite)
+    (DmlReadWrite)
+    (PgPerformHandling) 
+    (PGWaitingOnDocdb)
+    (PGActiveOnCPU)
```

Comment:
These events will come as part of PGGate and not TServer, is that fine?

Reply:
Yes. That was intentional.